### DISCUSSION TITLE
Welcome to Uyuni Discussions!

### DISCUSSION THREAD

**Discussion by renner** [uyuni-project/uyuni#2991] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mukul-kr** [comment]
Hey @renner I was wondering if you can provide me with any architecture diagram of uyuni. I was trying to understand where the downloading package feature mentioned in https://github.com/openSUSE/mentoring/issues/194 is present in the whole picture. A reference to the code file/folder for the same will also be very helpful.
